### DISCUSSION TITLE
Map keyboard Control+C => Command+C

### DISCUSSION THREAD

**Discussion by scharrin2** [utmapp/UTM#5819] [macOS]
Hello,

I am running a Windows 11 VM.

Is it possible to map the Mac keyboard combinations for Copy&Paste

Command+C => Control+C
Command+V => Control+V
Command+X => Control+X

Best regards,
Christian

**Comment by lgrachov** [comment]
You can use something like [Karabiner-Elements](https://karabiner-elements.pqrs.org/) to remap the keys.

**Reply by scharrin2** [reply]
I need something the other way round…

A software for Windows so that I can use the Mac-Keys in the Windows 11 machine.

**Reply by lgrachov** [reply]
> I need something the other way round…
> 
> 
> 
> A software for Windows so that I can use the Mac-Keys in the Windows 11 machine.

> This script should work with AutoHotkey 1.1 and later

The <kbd>Command</kbd> key is received in Windows as the <kbd>Win</kbd> (Windows) key. You can try using [AutoHotkey](https://www.autohotkey.com/) and rebinding the keys with this AutoHotkey script:
```
LWin::Ctrl
```

You can replace `LWin` with `RWin` if you want to. `LWin` is the left command and `RWin` is the right command.

**Comment by lgrachov** [comment]
> This script should work with AutoHotkey 1.1 and later

The <kbd>Command</kbd> key is received in Windows as the <kbd>Win</kbd> (Windows) key. You can try using [AutoHotkey](https://www.autohotkey.com/) and rebinding the keys with this AutoHotkey script:
```
LWin::Ctrl
```

You can replace `LWin` with `RWin` if you want to. `LWin` is the left command and `RWin` is the right command.

**Reply by scharrin2** [reply]
Thank you - I will try it
